compendium/modules/w06-patterns-exercise.tex

```diff
@@ -337,20 +335,23 @@
   case Färg.Ruter   => Färg.Hjärter
   case Färg.Klöver  => Färg.Spader
 \end{Code}
-
+Likt uppgift \ref{task:match-sealedtrait}\ref{subtask:match-sealedtrait-function} så kan även här en \code{import}-sats skrivas för att nå medlemmarna i \code{Färg} utan punktnotation.
+Det är dock inte alltid fördelaktigt att importera medlemmar till den globala namnrymden, då det kan förekomma namnkrockar. Anta ett exempel där vi jobbar på ett program med grafiskt användargränssnitt där vi har en färg \code{Red} definerad.
+Anta också att vi nu till vårt program vill importera ytterligare en röd färg för kulörerna hjärter och ruter, denna också namngiven \code{Red}. I detta scenario hade det uppstått en namnkrock då \code{Red} redan är definerad så importeringen hade ej kunnat ske.
 
 \SubtaskSolved
-\TODO %Se över denna
-Vid mönstermatchning så fungerar \code{sealed trait} ihop med \code{case}-objekt i praktiken likadant som att använda sig av \code{enum}. Vi såg att i deluppgift \ref{subtask:match-forgetcase} så varnade REPL redan vid kompilering att denna funktionen inte var uttömd \Eng{exhaustive}. Detta gäller även vid användning av \code{enum} också.
+Vid mönstermatchning så fungerar \code{sealed trait} ihop med \code{case}-objekt i praktiken likadant som att använda sig av \code{enum}. 
+Vi såg att i deluppgift \ref{task:match-sealedtrait}\ref{subtask:match-forgetcase} så varnade REPL redan vid kompilering att denna funktionen inte var uttömd \Eng{exhaustive}. Detta gäller även vid användning av \code{enum} också.
```

Comment:
change "denna funktionen inte var uttömd" till "denna matchning inte var uttömmande"  
remove "också" (redundant to "även")